### DISCUSSION TITLE
Not enough information in documentation

### DISCUSSION THREAD

**Discussion by ianthedev** [KingSora/OverlayScrollbars#665] [Q&A] [answered]
I initialized OverlayScrollbars on `document.body`. That worked.

But I also need to add a callback which should be dispatched after OverlayScrollbars scrolls the page to the top. The documentation severely lacks information in this respect.

I saw this code in the documentation:
```
OverlayScrollbars(document.querySelector('#myElement'), {}, {
  updated(osInstance, onUpdatedArgs) {
    // ...
  }
});
```

Then I wrote this in my code:

```
OverlayScrollbars(document.body, {}, {
  scroll(instance, event) {
    console.log(instance, event);
  }
});
```

After I scrolled my page, nothing happens. Not even an error.

Also, the [browser demo](https://codesandbox.io/p/devbox/overlayscrollbars-example-29hk3v) uses TyepScript. However, not everyone uses TypeScript. Such a demo is not helpful.

**Comment by KingSora** [comment, marked as answer]
@ianthedev thanks for the feedback!

Your example code should work, I've created an example for you: https://stackblitz.com/edit/vitejs-vite-vvnfw1?file=main.js&terminal=dev

I also want to point out that the `scroll` event is just directly forwarded from the native scroll event. If you want to register you own scroll events or any other scroll related listeners you can do it like this:

```
const osInstance = OverlayScrollbars(document.body, {});
const { scrollEventElement } = osInstance.elements();
scrollEventElement.addEventListener('scroll', () => {

});
```

I'll add an dedicated section for this topic into the FAQ section in the docs.
